graylog2-web-interface/src/components/pipelines/ProcessingTimelineComponent.tsx

```diff
@@ -119,20 +157,24 @@ const ProcessingTimelineComponent = () => {
       // TODO: Replace with ConfirmDialog components
       // eslint-disable-next-line no-alert
       if (window.confirm(`Do you really want to delete pipeline "${pipeline.title}"? This action cannot be undone.`)) {
-        PipelinesActions.delete(pipeline.id);
+        PipelinesActions.delete(pipeline.id).then(() => {
+          setPagination({ page: DEFAULT_PAGINATION.page, perPage, query });
+        });
       }
     };
   };
 
   const _pipelineFormatter = (pipeline) => {
+    const { id, title, description, stages: pStages } = pipeline;
```

Comment:
can we all it `pipelineStages` or `stages`? `pStages` has no real meaning out of context of this destructure